oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/process/ProcessServiceLabelRecord.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.skywalking.oap.server.core.analysis.manual.process;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.skywalking.oap.server.core.Const;
+import org.apache.skywalking.oap.server.core.analysis.MetricsExtension;
+import org.apache.skywalking.oap.server.core.analysis.Stream;
+import org.apache.skywalking.oap.server.core.analysis.metrics.Metrics;
+import org.apache.skywalking.oap.server.core.analysis.worker.MetricsStreamProcessor;
+import org.apache.skywalking.oap.server.core.remote.grpc.proto.RemoteData;
+import org.apache.skywalking.oap.server.core.storage.annotation.Column;
+import org.apache.skywalking.oap.server.core.storage.type.Convert2Entity;
+import org.apache.skywalking.oap.server.core.storage.type.Convert2Storage;
+import org.apache.skywalking.oap.server.core.storage.type.StorageBuilder;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+import static org.apache.skywalking.oap.server.core.source.DefaultScopeDefine.PROCESS_SERVICE_LABEL;
+
+/**
+ * Process have multiple labels, such as tag.
+ * {@link ProcessServiceLabelRecord} could combine them in the service level.
+ * It could help to quickly locate the similar process by the service and label.
+ */
+@Setter
+@Getter
+@Stream(name = ProcessServiceLabelRecord.INDEX_NAME, scopeId = PROCESS_SERVICE_LABEL,
+        builder = ProcessServiceLabelRecord.Builder.class, processor = MetricsStreamProcessor.class)
+@MetricsExtension(supportDownSampling = false, supportUpdate = false)
+@EqualsAndHashCode(of = {
+        "serviceId",
+        "label"
+})
+public class ProcessServiceLabelRecord extends Metrics {
+
+    public static final String INDEX_NAME = "process_service_label";
+    public static final String SERVICE_ID = "service_id";
+    public static final String LABEL = "label";
+
+    @Column(columnName = SERVICE_ID)
+    private String serviceId;
+    @Column(columnName = LABEL)
+    private String label;
```

Comment:
label or labels?

Reply:
I just flat the labels to multiple records(If there have two labels in the service, there would have two records). Do you think we need to combine them as a single record?

Reply:
It is fine with multiple rows, I just want you to be clear about this in the PR. Make sure my further reviews follow your design.